crux-docker/crux.edn

```diff
@@ -1,4 +1,4 @@
-{:crux/node-opts {:crux.node/topology ['crux.standalone/topology]
+{:crux/node-opts {:crux.node/topology '[crux.standalone/topology crux.http-server/module]
```

Comment:
not sure about module vs topology. A module can house multiple topologies?

Reply:
since #568 I've been calling these maps of components 'modules' (clarified definitions in that PR), but the standalone one existed beforehand, so I didn't want to break existing configurations - worth it?

Reply:
added #607